displays/cavity_display/frontend/fault_count_display.py

```diff
@@ -89,6 +89,15 @@ def __init__(self, lazy_fault_pvs=True):
         self.cm_combo_box.currentIndexChanged.connect(self.update_cavity)
         self.cav_combo_box.currentIndexChanged.connect(self.update_cavity)
 
+    def make_tlc_list(self) -> List[str]:
```

Comment:
I think this only needs to be done once, so maybe having it as a class variable instead of a method?

Reply:
You're right, I only need it once! 

I think I don't know enough about class variables. How do I turn a block of code into a class variable? Like, would I do a `fault_tlc_list = make_tlc_list()` in the FaultCountDisplay class outside of the init?

Reply:
kind of, I would say get rid of the method entirely and just have `fault_tlc_list = set(map(lambda d: d["Three Letter Code"], parse_csv()))` right under the class signature